CodeGeneration/Sources/generate-swiftbasicformat/BasicFormatFile.swift

```diff
@@ -142,6 +142,7 @@ let basicFormatFile = SourceFile {
         case (.asKeyword, .exclamationMark),
              (.asKeyword, .postfixQuestionMark),
              (.initKeyword, .postfixQuestionMark),
+             (.initKeyword, .leftParen),
```

Comment:
Little nit pick: can we sort those alphabetical?

Reply:
done